anago

```diff
@@ -1232,7 +1232,8 @@ get_build_candidate () {
   local testing_branch
   local branch_head
   
-  branch_head=$($GHCURL "$K8S" "GITHUB_API/commits/$RELEASE_BRANCH" |\
+  # Get the git commit sha of the release branch
+  branch_head=$($GHCURL "$K8S_GITHUB_API/commits/$RELEASE_BRANCH" |\
```

Comment:
we should start to prefer the form:
```suggestion
  branch_head=$($GHCURL "${K8S_GITHUB_API}/commits/${RELEASE_BRANCH}" |\
```
because it makes it very obvious what the intended variable name / expanded part is.

we've _mostly_ stuck to this for new / patched code in the main repo

Reply:
(also thanks for the fix!!)

Reply:
thank you ben, I'll try to clean this up later

Reply:
did a followup pr here: https://github.com/kubernetes/release/pull/808

Reply:
@cpanato thank you!